src/mono/mono/mini/interp/whitebox.c

```diff
@@ -143,10 +143,8 @@ determine_verbose_level (TransformData *td)
 		mono_method_desc_free (desc);
 		if (match)
 			return 4;
-	} else {
-		if (strcmp (method->name, name) == 0)
-			return 4;
-	}
+	} else if (strcmp (method->name, name) == 0)
+        return 4;
```

Comment:
Indent looks wrong?

Reply:
It is not wrong though

Reply:
the `return 4` line is using spaces instead of tabs.  Mono [uses 8-space tabs](https://www.mono-project.com/community/contributing/coding-guidelines/#indentation)